PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.FSharp/Azure.FSharp.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net461</TargetFramework>
+        <RootNamespace>JetBrains.ReSharper.Azure.FSharp</RootNamespace>
+        <AssemblyName>JetBrains.ReSharper.Azure.FSharp</AssemblyName>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="JetBrains.Rider.SDK" Version="$(RiderSDKVersion)" />
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\Azure.Psi\Azure.Psi.csproj" />
+        <ProjectReference Include="..\Azure.Daemon\Azure.Daemon.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="FSharp.Compiler.Service, Version=2020.3.0.0, Culture=neutral, PublicKeyToken=3099b8d9d20e74bf">
```

Comment:
@mfilippov / @auduchinok Right now, I have referenced assemblies from the F# plugin manually. Support for "full plugins" (back-end and front-end) seem hard to do.

Reply:
It's true. We don't have auto-reference for bundled plugins :( Maybe we should think about it.